### PR TITLE
Changes to compile tor 0.3.5.3-alpha

### PR DESCRIPTION
**Pull request by dballard**
Addressing issue #9 this is a proposal, the changes needed to make 0.3.5.3-alpha be supported

Granted support for an alpha version may not be desired at this moment, but this work should carry forward and be relevant to when 0.3.5.* stabilizes so I'm PRing this now so it can go quickly at your discretion.

Thanks!

### CODE REVIEW

File: external/Makefile [hunk 1]
```diff
@@ -253,6 +253,7 @@ tor/configure:
 
 tor/Makefile: tor/configure
 	cp config.sub tor
+	sed -i '14i #include <errno.h>' tor/src/lib/fs/freespace.c
```
Comment (goapunk) [inline external/Makefile]: Since this is merely a workaround for a bug in tor we should maybe report this and remove it once it has been fixed.

Comment (goapunk) [reply external/Makefile]: Tracking: https://trac.torproject.org/projects/tor/ticket/28286

Comment (goapunk) [reply external/Makefile]: There's a fix on its way (see the ticket) so I think we should wait for the next alpha and remove this?

Comment (dballard) [reply external/Makefile]: sounds good to me

**Comment by goapunk** [comment]
As @n8fr8 applied the necessary changes (afaik) I think we can close this MR and the related issue.

**Pull request closed without merging**